gsms/gsms.py

```diff
@@ -474,10 +471,10 @@ def main() -> None:
                 if has_app(sunshine_apps=sunshine_apps, name=name):
                     continue
 
-                # Increase count here to exclude some stuff
-                count += 1
-
                 cmd = application.find("StreamingCommandLine").text
+                if not cmd or short_name not in gfe_apps["metadata"]:
+                    continue
```

Comment:
This will have to be below the declaration of `short_name`, since it hasn't been defined yet.

Reply:
I have moved that in code... not sure how to see that here

Reply:
I guess I wasn't very clear. I only wanted the if condition moved.

Reply:
That's all I did move wasn't it? Not sure how to see latest diff here,.

Reply:
No, you also moved the cmd declaration.

https://github.com/LizardByte/GSMS/pull/141/files

Reply:
OK, I moved that back